cpp/src/text/bpe/load_merge_pairs.cu

```diff
@@ -135,6 +154,15 @@ std::unique_ptr<bpe_merge_pairs> load_merge_pairs_file(std::string const& filena
   return std::make_unique<bpe_merge_pairs>(std::move(input_column), stream, mr);
 }
 
+std::unique_ptr<bpe_merge_pairs> load_merge_pairs(cudf::strings_column_view const& merge_pairs,
+                                                  rmm::cuda_stream_view stream,
+                                                  rmm::mr::device_memory_resource* mr)
+{
+  CUDF_EXPECTS(!merge_pairs.is_empty(), "Merge pairs must not be empty");
+  CUDF_EXPECTS(!merge_pairs.has_nulls(), "Merge pairs may not contain nulls");
```

Comment:
Just to make sure, are docs updated after adding those checks?

Reply:
Yes, the are documented here: https://github.com/rapidsai/cudf/blob/1640c16898da96880e31eb1a5233091b18ee35a3/cpp/include/nvtext/byte_pair_encoding.hpp#L122